README.md

```diff
@@ -155,42 +174,52 @@ A wide variety of examples showcasing what Nutrient Web SDK can do can be found
-   - Prevent stray comment markers when someone opens a thread but no longer wants it. If no real text is entered and no comments are posted, the yellow highlight/marker is auto-removed when the editor closes, keeping pages tidy. Press Escape to instantly clear temporary markers while drafting
+
+  - Prevent stray comment markers when someone opens a thread but no longer wants it. If no real text is entered and no comments are posted, the yellow highlight/marker is auto-removed when the editor closes, keeping pages tidy. Press Escape to instantly clear temporary markers while drafting

```

Comment:
Great to see customization API being helpful here. Minor comment in the above example. The following code can be removed:

```ts
render: (params) => {
            // Use default editor UI by returning null
            return null;
          }
```

`render` is optional, if not provided it would use the default UI.